povm_toolbox/library/randomized_projective_measurements.py

```diff
@@ -57,6 +58,9 @@ def __init__(
                 different angles of each effect. I.e. its length equals two times the number of
                 PVMs (since we have 2 angles per PVMs). If 2D, it will have a new set of angles
                 for each qubit.
+            measurement_twirl : option to randomly twirl the measurements. For each single-qubit
+                projective measurement, twirling is equivalent to flipping the measurement. This
+                is equivalent to taking the opposite Bloch vector in the Bloch sphere representation.
```

Comment:
I think this explanation should emphasize that it is "equivalent to _randomly_ flipping the measurement" (at least along these lines). Right now, the explanation seems to suggest that the measurement always gets flipped.

Updating this here, needs to be propagated through all docstrings.